lightstep/resource_metric_condition.go

```diff
@@ -136,7 +136,7 @@ func getSpansQuerySchema() *schema.Schema {
 				},
 				"operator": {
 					Type:         schema.TypeString,
-					ValidateFunc: validation.StringInSlice([]string{"latency", "rate", "error_ratio"}, false),
+					ValidateFunc: validation.StringInSlice([]string{"latency", "rate", "error_ratio", "count"}, false),
```

Comment:
Should we just remove this validation altogether and rely on the validation in the LS SaaS?

Reply:
yeah that's a fair question... i think i wanna punt, cuz there's a fair number of places that do this and i'd prefer to be comprehensive if/when we change this paradigm. but agreed that there's not much upside (and a lot of downside) to duplicating this kind of validation logic across the public api and the terraform provider

Reply:
https://lightstep.atlassian.net/browse/LS-36604

Reply:
Not 100% confident in this, but I assume the ValidateFunc implementation would result in a more "ergonomic" error message to the user as Terraform itself will understand the semantics of the error better than opaque API response. That would be the argument for keeping the TF validation as well (but am not sure it is worth the added maintenance of a bit of duplicate code -- so I do not have a strong opinion here!).